lib/cookbook-release/changelog.rb

```diff
@@ -120,9 +121,11 @@ def markdown_priority_nodes
       end
       if not_nodes_only_commits.any?
         result << "*#{NON_NODES_ONLY}*\n" << not_nodes_only_commits.map { |c| c.to_s_markdown(false) }.join("\n") << "\n"
+        result << "*#{NODES_ONLY}*\n" << cl.select(&:nodes_only?).map { |c| c.to_s_markdown(false) }.join("\n") << "\n"
```

Comment:
This is going to display "Commits impacting only nodes:" even if empty; which is not coherent with the other sections (risky and non-risky ones)
Could you add a condition around it like `cl.size !=not_nodes_only_commits.size + risky_commits.size`?

Reply:
fixed